cmd/duffle/credential_generate.go

```diff
@@ -53,7 +53,7 @@ func newCredentialGenerateCmd(out io.Writer) *cobra.Command {
 			}
 			csName := args[0]
 
-			bun, err := loadBundle(bf, insecure)
+			bun, err := loadBundle(bf)
```

Comment:
Is the `insecure` flag definition used anywhere else?

Reply:
Good catch. I thought I had gotten all of em but this was the last one! Thanks. Updated.